src/routes/auth.mjs

```
@@ -0,0 +1,26 @@
+import { Router } from "express";
+import passport from "passport";
+
+const router = Router();
+
+router.post("/api/auth", passport.authenticate("local"), (req, res) => {
+    res.sendStatus(200);
+});
+
+router.get("/api/auth/status", (req, res) => {
+    console.log(`Inside /auth/status endpoint`);
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)